bom/application/pom.xml

```diff
@@ -256,6 +256,7 @@
 
             <!-- RESTEasy dependencies, imported as a BOM -->
             <!-- As RESTEasy also imports Jackson, we need to import its BOM after the Jackson BOM -->
+            <!-- As RESTEasy also imports Vert.x transitively, we need to import its BOM after the Vert.x BOM -->
```

Comment:
This could be removed now since we didn't move the import.

Reply:
Yeah, I'll clean everything if I can get CI to pass. If not I'll dig a bit more.